VnE/Source/Extensions/ChemPack/applyCls.py

```
@@ -17,21 +17,22 @@ def execute():
+            ga = (ga / 180) * np.pi
 
             sin = np.sin
             cos = np.cos
-            cot = lambda x: np.tan(x)**-1
-            csc = lambda x: np.sin(x)**-1
-
-            mat = np.array([[a*sin(be)*np.sqrt(1-(cot(al)*cot(be) - csc(al)*csc(be)*cos(ga))**2), 0, 0],
-                            [a*csc(al)*cos(ga) - a*cot(al)*cos(be), b*sin(al), 0],
-                            [a*cos(be), b*cos(al), c]])
-            mat = mat.transpose()
+            n = (cos(al) - (cos(ga) * cos(be))) / sin(ga)
```

Ruff (0.14.14)

[error] Local variable `n` is assigned to but never used

Remove assignment to unused variable `n`

(F841)